consensus/types/src/indexed_attestation.rs

```diff
@@ -116,11 +114,13 @@ impl<E: EthSpec> IndexedAttestation<E> {
         }
     }
 
-    pub fn to_electra(self) -> Result<IndexedAttestationElectra<E>, ssz_types::Error> {
-        Ok(match self {
+    pub fn to_electra(self) -> IndexedAttestationElectra<E> {
+        match self {
             Self::Base(att) => {
                 let extended_attesting_indices: VariableList<u64, E::MaxValidatorsPerSlot> =
-                    VariableList::new(att.attesting_indices.to_vec())?;
+                    VariableList::new(att.attesting_indices.to_vec())
+                        .expect("MaxValidatorsPerSlot must be >= MaxValidatorsPerCommittee");
+                // TODO: Add test after unstable rebase https://github.com/sigp/lighthouse/blob/474c1b44863927c588dd05ab2ac0f934298398e1/consensus/types/src/eth_spec.rs#L541
```

Comment:
Since all variables involved are compile time we can guarantee that this function never panics, no need to make it fallible

Reply:
`electra_attestation_changes` has been updated with latest unstable

Reply:
Test added in https://github.com/sigp/lighthouse/pull/5940/commits/cbb7c5d8f41c10d99aa84653c665ae5cc7a5f843